sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -815,13 +813,15 @@ private TelemetryItem createExceptionTelemetryItem(String errorStack, SpanData s
         } else {
             setOperationName(telemetryBuilder, span.getAttributes());
         }
-        setTime(telemetryBuilder, span.getEndEpochNanos());
+        setTime(telemetryBuilder, event.getEpochNanos());
         setSampleRate(telemetryBuilder, sampleRate);
 
+        // should this map the span attributes or the event attributes?
```

Comment:
or both? how does event get its attributes, copied from span? if not, just event.

Reply:
yeah, I think this should be event, but it has been like this forever, and so we need to think how/if we want to introduce potentially breaking change

I'd like to get out the urgent regression fix and we can think longer on this

Reply:
might want to put TODO to remind us for later.

Reply:
```suggestion
        // TODO (trask) should this map the span attributes or the event attributes?
```